libs/blocks/tabs/tabs.js

```diff
@@ -39,7 +39,9 @@ function changeTabs(e) {
 }
 
 function getStringKeyName(str) {
-  return str.trim().toLowerCase().replace(/[^\w ]/g, '').replace(/ +/g, '-');
+  // regex to omit all special characters except Japanese or Korean language characters
+  const regex = /[^\w\s\u3000-\u303F\u3040-\u309F\u30A0-\u30FF\u3130-\u318F\uAC00-\uD7AF]+/g;
+  return str.trim().toLowerCase().replace(regex, '').replace(/ +/g, '-');
```

Comment:
Should this be consistent with space vs whitespace regex?
```suggestion
  return str.trim().toLowerCase().replace(regex, '').replace(/\s+/g, '-');
```

Reply:
Good call. This is updated. Thanks